pkg/apis/serving/v1alpha1/framework_tensorflow_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2019 kubeflow.org.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/kubeflow/kfserving/pkg/constants"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	resource "k8s.io/apimachinery/pkg/api/resource"
+)
+
+func TestAcceptGoodRuntimeVersion(t *testing.T) {
```

Comment:
Should we follow the same testing style across the project? I think the best practice in golang for unit test is to use table testing (https://github.com/golang/go/wiki/TableDrivenTests).

Reply:
Yeah I'll update these.